sdk/CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.1.2 - 2023-MM-DD
+### Fixed
```

Comment:
```suggestion
### Fixed

```

Reply:
Why resolved ?

Reply:
Oh damn I fucked up the suggestion sorry

Reply:
Can you fix the spacing stuff pls

Reply:
Done